MPU/src/mpuCanHandlers.cpp

```diff
@@ -58,8 +58,12 @@ void bmsCurrentLimits_cb(const CAN_message_t &msg)
 void motorMotion_cb(const CAN_message_t &msg)
 {
     //angular motor speed is found at bytes 2 and 3
-    int16_t motorSpeed = ((msg.buf[3] << 8) | msg.buf[2]) / 10;
-    motorController.setMotorSpeed(motorSpeed);
+    int16_t motorSpeed = (msg.buf[3] << 8) | msg.buf[2];
+    if (motorSpeed >> 15 == -1) {
```

Comment:
This feels super unnecessary. I don't see why you can't cast the result to a signed integer via ```int16_t motorSpeed = (int16_t)((msg.buf[3] << 8) | msg.buf[2]);```

Reply:
I get the logic of checking if its negative by seeing the most significant bit but like there are signed integers for a reason so we don't have to do this

Reply:
We should try what you are suggesting. When we tried it I tried doing `(uint16_t)((msg.buf[3] << 8) | (uint16_t)msg.buf[2]);`

Reply:
I agree that what you are saying should work